.github/workflows/build-pr.yml

```diff
@@ -88,3 +89,45 @@ jobs:
           name: e2e-logs
           path: |
             logs-*.json        
+  
+  build_test_aks_e2e:
+    name: "Build and run e2e Test on AKS"
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    permissions:
+      id-token: write
+      contents: read
+    strategy:
+      matrix:
+        KUBERNETES_VERSION: ["1.24.6"]
```

Comment:
Any reason for only testing with `1.24.x`? Are there plans to expand this matrix?

Reply:
I was just trying to keep consistent with the `build_test_e2e` job. Just added `1.23.12`.